tests/langchain/test_langchain_model_export.py

```diff
@@ -2922,6 +2922,7 @@ def test_langchain_model_inject_callback_in_model_serving(
 ):
     # Emulate the model serving environment
     monkeypatch.setenv("IS_IN_DB_MODEL_SERVING_ENV", "true")
+    monkeypatch.setenv("MLFLOW_ENABLE_TRACE_IN_SERVING", "true")
```

Comment:
Was this already fixed?

Reply:
Yup! I thought it will disappear from diff as we merged the other PR, but maybe I need rebase.